sources/api/migration/migrator/src/main.rs

```diff
@@ -281,16 +348,47 @@ fn select_migrations<P: AsRef<Path>>(
 
 /// Given the versions we're migrating from and to, this will return an ordered list of paths to
 /// migration binaries we should run to complete the migration on a data store.
-// This separation allows for easy testing of select_migrations.
-fn find_migrations<P>(paths: &[P], from: &Version, to: &Version) -> Result<Vec<PathBuf>>
+/// This separation allows for easy testing of select_migrations.
+#[deprecated(since = "0.3.5", note = "for unsigned migrations.")]
+fn find_unsigned_migrations<P>(paths: &[P], from: &Version, to: &Version) -> Result<Vec<PathBuf>>
 where
     P: AsRef<Path>,
 {
     let mut candidates = Vec::new();
     for path in paths {
-        candidates.extend(find_migrations_on_disk(path)?);
+        candidates.extend(find_unsigned_migrations_on_disk(path)?);
     }
-    select_migrations(from, to, &candidates)
+
+    select_unsigned_migrations(from, to, &candidates)
+}
+
+fn get_current_version<P>(datastore_dir: P) -> Result<Version>
```

Comment:
It looks like this function was just moved without change?  Can we put it back, to reduce the review effort in the PR and in future investigations?

Reply:
> It looks like this function was just moved without change? Can we put it back, to reduce the review effort in the PR and in future investigations?

Maybe? It looks like line 106 is available. Hopefully if I paste it there then git won't show it as having been moved. We'll see.

Reply:
Addressed in [4f34544](https://github.com/bottlerocket-os/bottlerocket/compare/afc4b5d9ddfb7c6fb24f30abdaf6a5a000df222c..4f345446016e314c3da3f0855c92db5ca979f2a4)